packages/core/src/form-field/FormActivationIndicator.tsx

```diff
@@ -51,9 +51,10 @@ export interface FormActivationIndicatorProps
   hasIcon?: boolean;
 }
 
-export const FormActivationIndicator: React.FC<
-  FormActivationIndicatorProps
-> = ({ hasIcon, validationStatus }: FormActivationIndicatorProps) => {
+export const FormActivationIndicator = ({
```

Comment:
should we change all arrow functions to be named functions if no forwardRef?
Not saying you should do it as part of this PR, but as a general good practice rule

Reply:
I would encourage it yeah.